app/src/main/java/com/duckduckgo/app/onboarding/ui/page/extendedonboarding/ExtendedOnboardingFeatureToggles.kt

```diff
@@ -31,14 +39,73 @@ interface ExtendedOnboardingFeatureToggles {
     fun self(): Toggle
 
     @Toggle.DefaultValue(false)
-    @Experiment
     fun noBrowserCtas(): Toggle
 
     @Toggle.DefaultValue(false)
-    @Experiment
     fun privacyProCta(): Toggle
 
     @Toggle.DefaultValue(false)
     @Experiment
     fun highlights(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun testPrivacyProOnboardingCopyNov24(): Toggle
+
+    enum class Cohorts(override val cohortName: String) : CohortName {
+        CONTROL("control"),
+        PROTECTION("protection"),
+        PIR("pir"),
+        VPN("vpn"),
+    }
+
+    companion object {
+        const val EXPERIMENT_PREFIX = "test"
+    }
+}
+
+internal suspend fun ExtendedOnboardingPixelsPlugin.testPrivacyProOnboardingShownMetricPixel(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "dialogShown" }
+}
+
+internal suspend fun ExtendedOnboardingPixelsPlugin.testPrivacyProOnboardingPrimaryButtonMetricPixel(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "primaryButtonSelected" }
+}
+
+internal suspend fun ExtendedOnboardingPixelsPlugin.testPrivacyProOnboardingSecondaryButtonMetricPixel(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "secondaryButtonSelected" }
+}
+
+suspend fun FeatureTogglesInventory.onboardingExperiments(): Toggle? {
+    return this.getAllTogglesForParent("extendedOnboarding").firstOrNull {
+        it.featureName().name.startsWith(EXPERIMENT_PREFIX) && it.isEnabled()
+    }
+}
+
+@ContributesMultibinding(AppScope::class)
+class ExtendedOnboardingPixelsPlugin @Inject constructor(private val inventory: FeatureTogglesInventory) : MetricsPixelPlugin {
+
+    override suspend fun getMetrics(): List<MetricsPixel> {
+        val activeToggle = inventory.onboardingExperiments() ?: return emptyList()
+
+        return listOf(
+            MetricsPixel(
+                metric = "dialogShown",
+                value = "1",
+                toggle = activeToggle,
+                conversionWindow = listOf(ConversionWindow(lowerWindow = 0, upperWindow = Int.MAX_VALUE)),
```

Comment:
You probably don't want the upper window to be the max integer. I'd talk to DS but you probably want to define something closer in time like day 7.

Reply:
After checking with DS team, I'll change the conversionWindow to 0-1